jts-transaction-service/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:22
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /home
+
+ENV NARAYANA_PORT 4710
+ENV NARAYANA_VERSION 5.2.1.Final
+ENV NARAYANA_HOME /home/narayana
+ENV JACORB_LOG_LEVEL 3
+
+RUN dnf install -y java wget unzip \
```

Comment:
Maybe it would be a good idea to specify full version of Java, since we ship multiple versions in Fedora. It would be clear what's there.

Other idea - why not base on the `jboss-base-jdk` image? It's CentOS 7 (currently) with OpenJDK 7 or 8, (depending on the tag).